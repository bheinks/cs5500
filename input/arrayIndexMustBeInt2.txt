(* 
  Index expression must be of type integer
*)

program p;

var A : array [5..10] of integer;
    x : char;

begin
  A[x] := 99
end.

