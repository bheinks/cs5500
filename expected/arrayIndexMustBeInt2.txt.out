Line 11: Index expression must be of type integer
